chia/_tests/pools/test_pool_cmdline.py

```diff
@@ -1,20 +1,1050 @@
+                        ">=#pending_change": 1,  # any amount increase
+                        "pending_coin_removal_count": 1,
+                    },
+                    2: {
+                        "set_remainder": True,
+                    },
+                },
+                post_block_balance_updates={
+                    1: {
+                        "confirmed_wallet_balance": -1,
+                        "unconfirmed_wallet_balance": 0,
+                        ">=#spendable_balance": 1,
+                        ">=#max_send_amount": 1,
+                        "<=#pending_change": 1,  # any amount decrease
+                        "<=#pending_coin_removal_count": 1,
+                    },
+                    2: {
+                        "set_remainder": True,
+                    },
+                    3: {"init": True, "unspent_coin_count": 1},
+                },
+            )
+        ]
+    )
+
+    summaries_response = await wallet_rpc.get_wallets(WalletType.POOLING_WALLET)
+    assert len(summaries_response) == 2
+    wallet_id: int = summaries_response[-1]["id"]
+
+    await verify_pool_state(wallet_rpc, wallet_id, expected_state=expected_state)
+    return wallet_id
+
+
+async def create_new_plotnft(
+    wallet_test_framework: WalletTestFramework, self_pool: bool = False, second_nft: bool = False
+) -> int:
+    wallet_state_manager: WalletStateManager = wallet_test_framework.environments[0].wallet_state_manager
+    wallet_rpc: WalletRpcClient = wallet_test_framework.environments[0].rpc_client
+
+    our_ph = await wallet_state_manager.main_wallet.get_new_puzzlehash()
+
+    if self_pool:
+        await wallet_rpc.create_new_pool_wallet(
+            target_puzzlehash=our_ph,
+            pool_url="",
+            relative_lock_height=uint32(0),
+            backup_host="",
+            mode="new",
+            state="SELF_POOLING",
+            fee=uint64(0),
+        )
+    else:
+        await wallet_rpc.create_new_pool_wallet(
+            target_puzzlehash=our_ph,
+            pool_url="http://pool.example.com",
+            relative_lock_height=uint32(5),
+            backup_host="",
+            mode="new",
+            state="FARMING_TO_POOL",
+            fee=uint64(0),
+        )
+
+    if second_nft:
+        return await process_second_plotnft_create(
+            wallet_test_framework=wallet_test_framework,
+            expected_state=PoolSingletonState.SELF_POOLING if self_pool else PoolSingletonState.FARMING_TO_POOL,
+        )
+    else:
+        return await process_first_plotnft_create(
+            wallet_test_framework=wallet_test_framework,
+            expected_state=PoolSingletonState.SELF_POOLING if self_pool else PoolSingletonState.FARMING_TO_POOL,
+        )
+
+
+@pytest.mark.parametrize(
+    "wallet_environments",
+    [
+        {
+            "num_environments": 1,
+            "blocks_needed": [1],
+            "trusted": True,
+            "reuse_puzhash": False,
+        }
```

Comment:
I don't see why these would be irrelevant to the test, so I wouldn't recommend specifying them so that all parametrizations run.

Reply:
True for most other similar tests in this file as well

Reply:
reuse_puzhash fails - unclear it's it is specific to plotnfts. I think Trusted probably works though.

Reply:
Fixed and tests enabled